tensorflow_addons/activations/__init__.py

```diff
@@ -19,3 +19,4 @@
 from __future__ import print_function
 
 from tensorflow_addons.activations.sparsemax import sparsemax
+from tensorflow_addons.activations.gelu import gelu
```

Comment:
Please arrange them in alphabetical order :-)

Reply:
Ahh...thanks for pointing it out